bank2ynab.conf

```diff
@@ -328,6 +328,19 @@ Source Filename Pattern = [0-9]{8}-[0-9]{8}-umsatz
 Input Columns = skip,Date,skip,Memo,skip,skip,skip,skip,skip,skip,skip,Payee,skip,skip,Inflow,skip,skip
 Date Format = %d.%m.%y
 
+[DE Landesbank Berlin Amazon Kreditkarten-Banking]
+# KKB-Umsätze_20210111_0101.csv
+Use Regex For Filename = True
+# First line is the date
+# Second line is the headers
+Header Rows = 2
+Source CSV Delimiter = ;
+Source Filename Pattern = KKB-Umsätze_[0-9]{8}_[0-9]{4}
+# "Kreditkartennummer (teilmaskiert)";"Transaktionsdatum";"Buchungsdatum";"Händler (Name, Stadt & Land)";"Umsatzkategorie";"Betrag in Fremdwährung";"Einheit Fremdwährung";"Umrechnungskurs";"Betrag in Euro";"Amazon Punkte";"Prime Punkte"
+Input Columns = skip,skip,Date,Payee,Memo,skip,skip,skip,Outflow,skip,skip
+Date Format = %d.%m.%Y
+Plugin = LBBAmazon
```

Comment:
This is an easily-implemented change to the inflow/outflow logic. I propose that we include it in the general workflow rather than having a plugin. In line 210 of `bank_process.py`, just add an "or" for a negative sign.